src/main/java/com/pokegoapi/api/pokemon/Pokemon.java

```diff
@@ -267,6 +267,14 @@ public int getIndividualDefense() {
 	public int getIndividualStamina() {
 		return proto.getIndividualStamina();
 	}
+	
+	/**
+	 * Calculates the pokemons IV ratio
```

Comment:
I can't get checkstyle to run. If I need to change it I will. I can also start working on annotating more of the classes.